ui-tests/src/test/java/org/openmrs/reference/page/AllergyPage.java

```diff
@@ -71,4 +72,8 @@ public void removeNoKnownAllergy() {
     public String getPageUrl() {
         return "/allergyui/allergies.page";
     }
+
+    public void confirmCancelAllergy(){
```

Comment:
@Ndacyayisenga-droid is `confirmCancelAllergy()` a requirement for [RATEST-225](https://issues.openmrs.org/browse/RATEST-225)?

Reply:
cc @kdaud  @sherrif10